src/vs/workbench/contrib/positronPlots/browser/components/openInEditorMenuButton.tsx

```diff
@@ -0,0 +1,85 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (C) 2024 Posit Software, PBC. All rights reserved.
+ *  Licensed under the Elastic License 2.0. See LICENSE.txt for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import React, { useState, useCallback, useEffect } from 'react';
+
+import { IAction } from '../../../../../base/common/actions.js';
+import { localize } from '../../../../../nls.js';
+import { ICommandService } from '../../../../../platform/commands/common/commands.js';
+import { ActionBarMenuButton } from '../../../../../platform/positronActionBar/browser/components/actionBarMenuButton.js';
+import { AUX_WINDOW_GROUP_TYPE, ACTIVE_GROUP_TYPE, SIDE_GROUP_TYPE, AUX_WINDOW_GROUP, ACTIVE_GROUP, SIDE_GROUP } from '../../../../services/editor/common/editorService.js';
+import { PlotsEditorAction } from '../positronPlotsActions.js';
+
+interface OpenInEditorMenuButtonProps {
+	tooltip: string;
+	defaultGroup: number;
+	commandService: ICommandService;
+}
+
+interface OpenInEditorCommand {
+	editorTarget: AUX_WINDOW_GROUP_TYPE | ACTIVE_GROUP_TYPE | SIDE_GROUP_TYPE;
+	label: string;
+}
+
+// create an array of action ids with labels
+const openInEditorCommands: Array<OpenInEditorCommand> = [
+	{
+		'editorTarget': AUX_WINDOW_GROUP,
+		'label': localize('positron-editor-new-window', 'Open in new window')
+	},
+	{
+		'editorTarget': ACTIVE_GROUP,
+		'label': localize('positron-editor-new-tab', 'Open in editor tab')
+	},
+	{
+		'editorTarget': SIDE_GROUP,
+		'label': localize('positron-editor-new-tab-right', 'Open in editor tab to the right')
```

Comment:
I wonder if we could use similar verbiage to the File Explorer and hook into this preference: `workbench.editor.openSideBySideDirection`? Then the user can define whether the plot will open to the right or down based on the same preference as the File Explorer. Maybe something like "Open in editor tab to the side"?

Or maybe the user will want "to the Side" for plots to be different than the File Explorer?

Here's the context menu when right-clicking on a file in the File Explorer:
<img width="373" alt="image" src="https://github.com/user-attachments/assets/6c95199c-d508-41e7-861c-c8ba9c6d16b9" />